tests/unit/algorithms/action/adapters/mmaction/models/recognizers/test_action_movinet_recognizer.py

```diff
@@ -0,0 +1,67 @@
+"""Unit Test for otx.algorithms.action.adapters.mmaction.models.recognizers.movinet_recognizer."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+from copy import deepcopy
+
+import pytest
+import torch
+from mmaction.models.recognizers.recognizer3d import Recognizer3D
+
+from otx.algorithms.action.adapters.mmaction.models.recognizers.movinet_recognizer import (
+    MoViNetRecognizer,
+)
+from tests.test_suite.e2e_test_system import e2e_pytest_unit
+
+
+class MockOTXMoViNet:
+    pass
+
+
+class MockModule:
+    def __init__(self):
+        self.backbone = MockOTXMoViNet()
+        self._state_dict = {
+            "classifier.0.conv_1.conv3d.weight": torch.rand(1, 1),
+            "conv1.conv_1.conv3d.weight": torch.rand(1, 1),
+        }
+        self.is_export = False
+
+    def state_dict(self):
+        return self._state_dict
+
+
+class TestMoViNetHead:
```

Comment:
I think this should be TestMoViNet. If I am wrong, just resolve this.

Reply:
Thanks for good catch, I'll rename to `TestMoViNetRecognizer`.